ext/dom/php_dom.h

```diff
@@ -146,7 +141,7 @@ entry = zend_register_internal_class_ex(&ce, parent_ce);
 }
 
 #define DOM_NO_ARGS() \
-	if (zend_parse_parameters_none() == FAILURE) { \
+	if (zend_parse_parameters_none() == failure) { \
```

Comment:
Why this change?

Reply:
Honestly, I have no idea how this change ended up being here. Maybe I unintentionally pressed a key combo which lowercased these characters? I'l get rid of this then.